dali/imgcodec/image_decoder.cc

```diff
@@ -219,6 +222,8 @@ class ImageDecoder::DecoderWorker {
   std::shared_ptr<ImageDecoderInstance> decoder_;
   bool produces_gpu_output_ = false;
 
+  std::string nvtx_marker_str_;
```

Comment:
Tiny nitpick - I'd move it where all the bookkeeping stuff is (i.e. after the `started_` flag).